tests/engine/test_stats.py

```diff
@@ -16,6 +15,16 @@
 HOR_DIM_SIZE = 100
 HEIGHT_DIM_SIZE = 5
 
+TIME_DIM_GRIB_SIZE = 1
+STEP_DIM_SIZE = 1
+HEIGHT_DIM_GRIB_SIZE = 1
+HORIZONTAL_DIM_GRIB_SIZE = 6114
+
+GRIB_FILENAME = "test_stats_grib.grib"
```

Comment:
```suggestion
GRIB_FILE_NAME = "test_stats_grib.grib"
```

Reply:
@huppd Pinging just in case you missed this comment.

Reply:
thanks, should be fixed by now.

Reply:
upps, forgot to push.

Reply:
:ok_hand: